chainerrl/links/noisy_linear.py

```diff
@@ -26,10 +28,22 @@ def __init__(self, mu_link, sigma_scale=0.4):
 
         with self.init_scope():
             self.mu = mu_link
+            self.mu.W.initializer = VarianceScalingUniform(1)
+            if not self.nobias:
+                self.mu.b.initializer = VarianceScalingUniform(1)
```

Comment:
The bias parameter `b` is usually initialized as `0`.  On `mu.b`, don't you think it's also reasonable?

Reply:
That is true.